cern.root.ROOT.yaml

```
@@ -0,0 +1,26 @@
+app-id: cern.root.ROOT
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: root
+rename-desktop-file: root
+rename-icon: Root6Icon #128x128
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --filesystem=xdg-documents
+modules:
+  - name: root
+    buildsystem: cmake-ninja
+    builddir: true
+    post-install:
+      - install -Dm644 cern.root.ROOT.appdata.xml /app/share/appdata/cern.root.ROOT.appdata.xml
```

```suggestion
      - install -Dm644 ../cern.root.ROOT.appdata.xml /app/share/appdata/cern.root.ROOT.appdata.xml
```

since builddir is true